### ISSUE TITLE
How to launch and change topics for monocular visual inertial odometry?

### ISSUE THREAD

**Issue by nicko369** [rpng/open_vins#345]
I'm trying to run openvins together with a simple drone on ROS 
I'm just using a standard ROS camera plugin. Which configuration do I use?
How can I change the topics the openvins node is subscribed to as well?

**Comment by nicko369** [comment]
Also since I'm simuating this in gazebo, do I set the camera parameters and IMU parameters to 0?

**Comment by goldbattle** [comment]
If you could provide more details and configuration you are using, that would be great. Make sure to go through the tutorials to understand the package first and what is required as input (inspect the example rosbags).
https://docs.openvins.com/gs-tutorial.html

You need some noises, so you could leave those at 1pixel and maybe the default for eurocmav. But you will need the calibration transform between the camera and IMU sensor you subscribe to.

**Comment by nicko369** [comment]
I've been going over the documentation page but to be honest it's not very intuitive.
I'm running openvins through dockers.
I've used both ros1 and ros2 dockerfiles you have provided however I can't even get roscore to start. 

I'd like to run this with a turtlebot waffle pi so I'd be using a pi cam.  
I understand I'd need at least the image_raw and IMU data but I've renamed those topics which is why I'd like to remap them in openvins
